### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by joshprice**
- [ ] Needs a short intro https://github.com/gausby/gen_mqtt/pull/10/files#diff-04c6e90faac2675aa89e2176d2eec7d8R10

### CODE REVIEW

File: lib/gen_mqtt.ex [hunk 1]
```diff
@@ -25,18 +26,22 @@ defmodule GenMQTT do
       end
 
   This will log to the console every time a sensor posts a temperature
-  to the broker.
+  to the `room/<location/temp` topic.
+
+  It assumes there is an MQTT server running on localhost on port 1883.
```
Comment (gausby) [inline lib/gen_mqtt.ex:31]: Perhaps it should be specified, that it is the example that assume a server running on localhost port 1883; It is possible to specify server and port using configuration options.

Comment (joshprice) [inline lib/gen_mqtt.ex:31]: An example of setting some basic options might be handy

**Comment by gausby** [comment]
Do you have any good examples of project intros?

For GenMQTT I am currently thinking: Create processes that connect to, and communicate with, MQTT servers and fits snuggly into your OTP.

**Comment by gausby** [comment]
I will close this PR…sorry that we didn't get it in.

I will start work on another client. Let's talk if you wanna help out with the implementation and documentation :)

**Pull request closed without merging**